lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -973,7 +973,7 @@ public function getDropIndexSQL($index, $table=null)
             return $this->getDropPrimaryKeySQL($table);
         }
 
-        return 'DROP INDEX ' . $indexName . ' ON ' . $table;
+        return 'DROP INDEX `' . $indexName . '` ON ' . $table;
```

Comment:
Then do i have to modify the line there:

return 'DROP INDEX $this->quoteStringLiteral()' . $indexName . '$this->quoteStringLiteral() ON ' . $table;